### PR TITLE
[WebNN EP] Remove activation fusion

### PR DESCRIPTION
**Pull request by Honry**
WebNN spec has removed activation option for conv and batchNormalization. We don't need additional activation fusion in WebNN EP anymore.

[edit by fdwr] Note this is handled in the browser now, which knows more about the backend platform version and can more safely make decisions about which fusions are possible (e.g. for the DirectML backend, whether softmax and gelu can fuse successfully with their base operator).

### CODE REVIEW

**Comment by Honry** [comment]
@fdwr, @guschmue, @huningxin, PTAL, thanks!

**Review by huningxin — APPROVED** [review]
LGTM, thanks!

**Review by fdwr — APPROVED** [review]
👍

**Comment by Honry** [comment]
@fdwr, there's one unexpected CL failure, could you help restart it?

**Comment by Honry** [comment]
@fdwr, aha, pass now! Thanks!
